From 8a2b50ed10cc73aa55153584995a94f9467c1063 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Fri, 23 Sep 2016 11:53:41 +0200
Subject: [PATCH] topic-miami: Enable second ethernet controller

Configure the ethernet GEM1 to use the EMIO interface. This allows
designs that use the second controller to actually work.

Signed-off-by: Mike Looijmans <mike.looijmans@topic.nl>
---
 board/topic/zynq/zynq-topic-miami/ps7_init_gpl.c     | 2 ++
 board/topic/zynq/zynq-topic-miamiplus/ps7_init_gpl.c | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/board/topic/zynq/zynq-topic-miami/ps7_init_gpl.c b/board/topic/zynq/zynq-topic-miami/ps7_init_gpl.c
index ebc6924..9a5ded2 100644
--- a/board/topic/zynq/zynq-topic-miami/ps7_init_gpl.c
+++ b/board/topic/zynq/zynq-topic-miami/ps7_init_gpl.c
@@ -40,7 +40,9 @@ static unsigned long ps7_clock_init_data_3_0[] = {
 	EMIT_MASKWRITE(0XF8000008, 0x0000FFFFU, 0x0000DF0DU),
 	EMIT_MASKWRITE(0XF8000128, 0x03F03F01U, 0x00302301U),
 	EMIT_MASKWRITE(0XF8000138, 0x00000011U, 0x00000011U),
+	EMIT_MASKWRITE(0XF800013C, 0x00000011U, 0x00000011U),
 	EMIT_MASKWRITE(0XF8000140, 0x03F03F71U, 0x00100141U),
+	EMIT_MASKWRITE(0XF8000144, 0x03F03F71U, 0x00100141U),
 	EMIT_MASKWRITE(0XF8000148, 0x00003F31U, 0x00000C01U),
 	EMIT_MASKWRITE(0XF800014C, 0x00003F31U, 0x00000601U),
 	EMIT_MASKWRITE(0XF8000150, 0x00003F33U, 0x00001803U),
diff --git a/board/topic/zynq/zynq-topic-miamiplus/ps7_init_gpl.c b/board/topic/zynq/zynq-topic-miamiplus/ps7_init_gpl.c
index 1a493f6..be346b7 100644
--- a/board/topic/zynq/zynq-topic-miamiplus/ps7_init_gpl.c
+++ b/board/topic/zynq/zynq-topic-miamiplus/ps7_init_gpl.c
@@ -40,7 +40,9 @@ static unsigned long ps7_clock_init_data_3_0[] = {
 	EMIT_MASKWRITE(0XF8000008, 0x0000FFFFU, 0x0000DF0DU),
 	EMIT_MASKWRITE(0XF8000128, 0x03F03F01U, 0x00302301U),
 	EMIT_MASKWRITE(0XF8000138, 0x00000011U, 0x00000011U),
+	EMIT_MASKWRITE(0XF800013C, 0x00000011U, 0x00000011U),
 	EMIT_MASKWRITE(0XF8000140, 0x03F03F71U, 0x00100141U),
+	EMIT_MASKWRITE(0XF8000144, 0x03F03F71U, 0x00100141U),
 	EMIT_MASKWRITE(0XF8000148, 0x00003F31U, 0x00000C01U),
 	EMIT_MASKWRITE(0XF800014C, 0x00003F31U, 0x00000601U),
 	EMIT_MASKWRITE(0XF8000150, 0x00003F33U, 0x00001803U),
-- 
1.9.1

